src/ts/components/transaction/Confirm.tsx

```diff
@@ -0,0 +1,257 @@
+    el.value = val
+    el.setAttribute('readonly', '')
+    el.style.position = 'absolute'
+    el.style.left = '-9999px'
+    document.body.appendChild(el)
+    el.select()
+    document.execCommand('copy')
+    document.body.removeChild(el)
+
+    this.setState({ message: t('copyAddressMessage') })
+    const timeout = setTimeout(() => {
+      this.setState({ message: '' })
+    }, 100)
+    this.setState({ msgTimeout: timeout })
+  }
+
+  render () {
+    return (
+    <Modal trigger={this.props.trigger} style={{ 'zIndex': 3 }}>
+      <UpperSection>
+        <Offset>
+          <Status>
+            <Identicon account={this.props.fromAddress} size={48}/>
+          </Status>
+        </Offset>
+        <Heading>Confirm Extrinsic</Heading>
+        <Subheading>Review your extrinsic details</Subheading>
+      </UpperSection>
+      <OverlaySection>
+        <Overlay/>
+      </OverlaySection>
+      <Section>
+        <FromTo color={colorSchemes[this.props.color].backgroundColor}>
+          <Icon name='arrow circle right' size={'big'} style={{  'marginLeft': '10px' }}/>
+          <Container textAlign={'left'} style={{ 'marginLeft': '10px' }}>
+            <Tooltip
+              title={!this.state.message ? t('copyToClipboard') : t('copiedExclam')}
+              position='bottom'
+              trigger='mouseenter'
+              arrow={true}
+            >
+              <span onClick={() => this.copyToClipboard(this.props.toAddress)}>{this.truncate(this.props.toAddress)}</span>
+            </Tooltip>
+          </Container>
+        </FromTo>
+      </Section>
+      <Section>
+        <Info>
+          <Key>Fee</Key>
+          <Value>{formatBalance(this.props.fee)}</Value>
+        </Info>
+      </Section>
+      <Section>
+        <Info>
+          <Key>Amount</Key>
+          <Value>{formatBalance(this.props.amount)}</Value>
+        </Info>
+      </Section>
+      <Section>
+        <Info>
+          <Key>total</Key>
+          <Value>{formatBalance(this.props.amount.add(this.props.fee))}</Value>
+        </Info>
+      </Section>
+      <Section>
+        <Info>
+          <Warning>
+            <div>
+              <Icon name='warning sign' size={'small'}/>
+              The final recipient balance is less or equal to {formatBalance(this.props.existentialDeposit)} (the existential amount) and will not be reflected
+            </div>
+            <div>
+              <Icon name='warning sign' size={'small'}/>
+              A fee of {formatBalance(this.props.creationFee)} will be deducted from the sender since the destination account does not exist
+            </div>
+            <div>
+              <Icon name='warning sign' size={'small'}/>
+              Fees include the transaction fee and the per-byte fee
+            </div>
+          </Warning>
+        </Info>
+      </Section>
+      <Section style={{ 'marginTop': '20px' }}>
+        <Info>
+          <Button>Cancel</Button>
```

Comment:
we probably should use translate for all texts, it will make our internalization work easier later.